src/index.html

```diff
@@ -1,13 +1,453 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link rel="icon" href="./images/favicon.svg">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section page__section--1 header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.png"
+                alt="MyBike"
+              >
+            </a>
+
+            <a
+            href="tel:+1 234 555-55-55"
+            class="icon icon--phone"
+            >
```

Comment:
```suggestion
            <a
              href="tel:+1 234 555-55-55"
              class="icon icon--phone"
            >
```
Check your code formatting everywhere
https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping